src/convertToObject.js

```diff
@@ -12,7 +12,19 @@
  * @return {object}
  */
 function convertToObject(sourceString) {
-  // write your code here
+  const convertStylesToArr = sourceString
+    .split('\n')
+    .filter(prop => prop.includes(':'));
+  const stylesObj = {};
+
+  convertStylesToArr.forEach(el => {
+    const style = el.slice(0, el.lastIndexOf(';')).split(':');
+    const [key, value] = [style[0].trim(), style[1].trim()];
+
+    stylesObj[key] = value;
+  });
+
+  return stylesObj;
```

Comment:
let's separate concerns here
fully prepare your array inside convertStylesToArr variable (split, trim, filter and etc), so then in forEach loop you will only need to assign values to an object